client/src/utils/parser/robotCodeToSsotParsing/robotCodeToSsotParsing.test.js

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable import/first */
+import { correctSettingsSection, correctTaskSection, taskAndApplicationCombinations, correctElementsArray, correctInstructionBlocks } from './robotCodeTestSnippets';
+
+jest.mock('../../notificationUtils');
+
+import customNotification from '../../notificationUtils';
+
+
+const parser = require('./robotCodeToSsotParsing');
+
+const ROBOT_ID = '54ab2d30eb3cc402041ac60f';
+
+sessionStorage.setItem('robotName', 'AwesomeTestRobot')
+sessionStorage.setItem('robotId', '12345678')
+sessionStorage.setItem('availableApplications', '["Excel.Application","Excel.Files","HTTP"]')
+sessionStorage.setItem('TaskApplicationCombinations', JSON.stringify(taskAndApplicationCombinations))
+
+const correctRobotCode = `${correctSettingsSection}\n\n${correctTaskSection}`
+const robotCodeAsArray = parser.getRobotCodeAsArray(correctRobotCode);
+const settingsSectionAsArray = parser.getRobotCodeAsArray(correctSettingsSection);
+const taskSectionAsArray = parser.getRobotCodeAsArray(correctTaskSection);
+const declaredApplications = parser.getApplicationArray(settingsSectionAsArray);
+
+describe('RobotCode to SSOT Parsing Tests', () => {
+    test('ssot contains right robotName', () => {
+        const ssot = parser.parseRobotCodeToSsot(correctRobotCode);
+        return expect(ssot).toHaveProperty('robotName', 'AwesomeTestRobot');
```

Comment:
You don't return anything from test methods

Reply:
Thanks for the hint. It's fixed now